pkg/antctl/raw/multicluster/delete/clusterset.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package delete
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type clusterSetOptions struct {
+	namespace string
+}
+
+var clusterSetOpt *clusterSetOptions
+
+var clusterSetExamples = strings.Trim(`
+Delete a ClusterSet in Kubernetes cluster in a specified Namespace
```

Comment:
What does "in Kubernetes cluster" mean?

And I am totally lost what commands run in leader and what run in member. Could you add some code comments for that?

Reply:
"in Kubernetes cluster" means a leader or member cluster. I changed it to the "a leader or member cluster."